setup.py

```diff
@@ -12,6 +12,7 @@
     "napari-ndtiffs",
     "brainglobe-napari-io",
     "cellfinder-core>=0.2.4",
+    "pooch",  # For downloading sample data
```

Comment:
Should we pin the version?

Reply:
I went for `>1` because I guess that's when the API stabilised.